news/2018-07-17-new-featured-artists-antiplur-and-nakanojojo.md

```diff
@@ -0,0 +1,91 @@
+---
+layout: post
+title: "New Featured Artists: antiPLUR & Nakanojojo"
+date: 2018-07-17 12:00:00 +0000
+---
+
+We're proud to announce the addition of **two** new Featured Artists to the roster, **antiPLUR** and **Nakanojojo**! Check them out!
+
+![](/wiki/shared/news/banners/featured-artist.jpg)
+
+A departure from the norm, this time we've got two brand new artists to share with you. Each one currently has **four** tracks available on their listing, for a total of eight new tracks altogether.
```

Comment:
listings

Reply:
their listings implies they have multiple of them, which they don't, they've got one each